monitoring/prometheus.md

```diff
@@ -40,6 +40,11 @@ Use our template to define a ServiceMonitor and to create a service account and
 oc process -f fuse-servicemonitor.yml -p NAMESPACE=<your-fuse-namespace> -p FUSE_SERVICE_NAME=<fuse-app-name> | oc apply -f -
 ```
 
+>**NOTE:** Make sure the configuration of the created service's selector is correct and matches all conditions properly.
```

Comment:
What does this entail?

Reply:
@cunningt in yaml file of a service there is a selector and in my case fuse console quickstarts didn't have _app: name_ labels but the selector was relying on it. Therefore, the metrics weren't exposed. I added this note to check it carefully whether selector is configured correctly. Otherwise, it won't be working. Perhaps, I have to make it more clear.